### ISSUE TITLE
darknet2pytorch.py文件中有些地方不是很明白

### ISSUE THREAD

**Issue by oreo-lp** [Tianxiaomo/pytorch-YOLOv4#44]
tool文件夹下的darknet2pytorch.py的169行紧跟在continue语句的下面，那这些代码是不是不会被执行？
`if self.loss:`
`self.loss = self.loss + self.models[ind](x)`
`else:`
`self.loss = self.models[ind](x)`
上面这段代码是不是应该放在176行yolo层下面？

**Comment by Tianxiaomo** [comment]
没用到

**Comment by feixiangdekaka** [comment]
> 没用到

请问如何装换权重这个文件好像不能转换pt  weights
